cmake/Modules/FindMKL.cmake

```diff
@@ -2,6 +2,8 @@
 #
 # Options:
 #
+#   USE_MKLML_MKL                   : Search for MKL:ML library variant
```

Comment:
Added a subsection option to use MKLML (2017) MKL. Otherwise, it uses the old logic.